### PR TITLE
Logging Cosmetics

### PR DESCRIPTION
**Pull request by dxps**
Hello,

I've added some logging cosmetics, including the current execution thread name and id to illustrate the fact that the event driven services are executed on different threads, therefore you get parallelism by default.

The result is the following:
image

As the hiring effects are triggered and executed in parallel threads, the order of the entries in the console may change from one execution to the other.

### CODE REVIEW

**Comment by dxps** [comment]
Well, this is a primitive way of getting the thread id.

Meanwhile, I changed a little bit the approach:
- replaced any `System.out.print*()` calls with SLF4J's `logger.info()`
- included the `threadId` in the logback format with the help of a simple <a href="https://github.com/vision8tech/qbit-event-bus-cluster-sample/blob/master/src/main/java/tech/vision8/reactives/qbit/eventbus/cluster/sample/utils/ThreadIdConverter.java" target="_blank">converter</a>.
```xml
<conversionRule
  conversionWord="threadId"
  converterClass="tech.vision8.reactives.qbit.eventbus.cluster.sample.utils.ThreadIdConverter"/>
```

Long story short, if you don't like this PR I can submit a new PR with the aforementioned version. 😃
